src/schemas/maps/schema.ts

```diff
@@ -6,7 +6,7 @@ import { mapValues } from '@/utilities/object'
 
 export const mapSchemaResponseToSchema: MapFunction<SchemaResponse, Schema> = function(source) {
   return {
-    definitions: source.definitions,
+    definitions: this.map('SchemaPropertiesResponse', source.definitions, 'SchemaProperties'),
```

Comment:
oh i'm surprised we weren't already doing this

Reply:
Same, but this really only matters for block definitions which we're not really using.